src/containers/downloads/DownloadCasesModal.js

```diff
@@ -30,8 +30,8 @@ type Props = {
 const DownloadCasesModal = ({ isVisible, onClose, staff } :Props) => {
 
   const [selectedStaffMember, selectStaffMember] = useState('');
```

Comment:
could you also rename this setter to `setSelectStaffMember`?

Reply:
ah yep!

Reply:
done!